project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -93,6 +98,58 @@ Mono<ProjectCreationResponse> create(ProjectCreationRequest request)
       throws UnknownRequestException, RequestFailedException, AccessDeniedException;
 
 
+  /**
+   * Requests {@link SamplePreview} for a given experiment.
+   *
+   * @param experimentId the experiment ID for which the sample preview shall be retrieved
+   * @return a reactive stream of {@link SamplePreview} objects of the experiment
+   * @throws RequestFailedException if the request could not be executed
+   * @since 1.10.0
+   */
+  Flux<SamplePreview> getSamplePreviews(String experimentId) throws RequestFailedException;
```

Comment:
You will need the project identifier as well as we need to make sure that the access rights are correct. This holds true for all api methods